lib/builder.js

```diff
@@ -111,7 +112,8 @@ Builder.prototype.build = function (willReadStringTree) {
       name: getDescription(tree),
       broccoliNode: true,
       broccoliId: node.id,
-      broccoliCachedNode: false
+      broccoliCachedNode: false,
+      broccoliPluginName: getPluginName(tree)
```

Comment:
why do we separate `broccoliPluginName` from `name`?  seems reasonable just curious